src/_core.scss

```diff
@@ -0,0 +1,14 @@
+// GLOBAL SCOPE LOADER: CORE
+// This partial loads Core scope related Partials.
+// - This partials Generates Core custom properties using a mixin and a settings map.
+//   - Mixin is "generate-custom-properties".
+//   - Map is "$core-settings-map" which is a map collected from all other core scope maps.
```

Comment:
why the name of this file starts with _

Reply:
scss convention for a partial.
In scss we have two conventions 
* plain name that indicates a loader, this means this file should be directly output.
* name that starts with "_" this means it we be imported and not directly compiled.